ibexa/connector-qualifio/5.0/config/packages/ibexa_connector_qualifio.yaml

```diff
@@ -43,3 +43,8 @@
 #                id: 'userid'
 #                remote_id: 'identifier'
 #                login: 'login'
+
+ibexa_fieldtype_richtext:
+    enabled_attribute_types:
+        - campaign
+        - measurement
```

Comment:
Why is `measurement` enabled as part of `ibexa_connector_qualifio.yaml` package config?